docs/diagnostics/PX1036.md

```diff
@@ -5,20 +5,38 @@ This document describes the PX1036 diagnostic.
 
 | Code   | Short Description                                 | Type  | Code Fix    | 
 | ------ | ------------------------------------------------- | ----- | ----------- | 
-| PX1036 | The DAC primary key should be named `PK`. The class containing DAC foreign keys should be named `FK`.  | Warning (ISV Level 3: Informational) | Available | 
+| PX1036 | The DAC must have one primary key which should be named `PK`. The class containing DAC foreign keys should be named `FK`. The single unique key in the DAC should be named `UK`. All unique keys in the DAC should be declared in a public static class named `UK`. | Warning (ISV Level 3: Informational) | Available | 
 
 ## Diagnostic Description
-The diagnostic checks the correct naming of a primary and foreign key field declarations in DACs. For a primary key, the name should be `PK`. For a foreign key, the name should be `FK`.
+The diagnostic checks the correct naming of a primary, foreign, and unique key field declarations in DACs. For a primary key, the name should be `PK`. For a foreign key, the name should be `FK`. For a unique key, the name should be `UK`.
 The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
 The diagnostic does not check abstract DACs and DACs which have only unbound DAC properties (fully-unbound DACs used for custom popups and filters for inquiry forms).
 
-To fix the issue, change the name of the primary key class to `PK`, and change the name of the class that contains foreign key classes to `FK`.
+The diagnostic also checks the declaration and implementation of a unique key by the following rules:
+ - A single unique key in a DAC declaration must have the name `UK`.
+ - Multiple unique keys can have arbitrary names but they must be declared inside a public static class named `UK`.
+ - Primary and unique key declarations cannot have the same set of fields.
+ - There cannot be a unique key declaration without a primary key declaration. You should turn at least one unique key into a primary key. Acuminator searches for an appropriate candidate for a primary key amoung the DAC key fields by the following rules:
+   - If there is one key field in the DAC, Acuminator suggests to turn it into a primary key named `PK` by renaming and optionally changing its location to make it a first DAC's type member. This is actual for unique keys which are not direct DAC type members because they are declared in the `UK` container class.
```

Comment:
```suggestion
   - If there is a suitable unique key in the DAC, Acuminator suggests to turn it into a primary key named `PK` by renaming it and optionally changing its location to make it a first DAC's type member. This is actual for unique keys which are not direct DAC type members because they are declared in the nested `UK` container class.
```